es2017.js

```diff
@@ -51,7 +51,12 @@ var ES2017 = assign(assign({}, ES2016), {
 
 	// https://www.ecma-international.org/ecma-262/8.0/#sec-iterabletolist
 	IterableToList: function IterableToList(items, method) {
-		var iterator = this.GetIterator(items, method);
+		var iterator;
+		if (arguments.length < 2) {
```

Comment:
this change is part of the as-yet unpublished ES2020; and should never be made in the ES2017 method.

I’ll take care of adding it after it’s finalized in a few months.

Reply:
But what about: https://github.com/ljharb/es-abstract/blob/4539827e66ad8406bf3b6fd44daf7fb491bbbfc6/es2015.js#L1021-L1022

Isn’t this a similar situation?

Reply:
No, because in that one, the unfixed algorithm is actually so broken it’s useless.

In this case, it’s just a changed method signature.